morph_tool/resampling.py

```diff
@@ -1,20 +1,19 @@
-"""Resampling for morphio morphologies
+"""Resampling functionality for morphio morphologies. It allows for the
+generation of new points on the morphology skeleton with different frequency
```

Comment:
frequency: did you mean "density" (i.e., number of points per micron)?

Reply:
fixed